pkgs/development/python-modules/bpython/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, buildPythonPackage, fetchurl, pygments, greenlet, curtsies, requests, mock }:
+
+buildPythonPackage rec {
+  pname = "bpython";
+  version = "0.17";
+  name  = "${pname}-${version}";
+
+  # 0.17 is still missing on PyPI, https://github.com/bpython/bpython/issues/706
+  src = fetchurl {
```

Comment:
fetchPypi

Reply:
0.17 is not released to PyPI yet, thats why `fetchUrl` is used. Issue is linked above.